src/renderer/src/scenes/main-scene/styles.css

```diff
@@ -1,4 +1,14 @@
 .main-scene {
+
+  .song-detail-gradient {
```

Comment:
You can remove this gradient, we are not using gradients on the main scene anymore.